Doc/Manual/SWIGPlus.html

```diff
@@ -1829,6 +1829,7 @@ <H2><a name="SWIGPlus_default_args">6.14 Default arguments</a></H2>
 which don't have optional arguments in the language, 
```

Comment:
This line needs correcting in your additions to this file should be put here.

Reply:
I took a guess at what I thought you meant in my second commit, if I still have it all wrong then I think I need more explicit instructions on what goes where.